registration/include/pcl/registration/impl/default_convergence_criteria.hpp

```diff
@@ -107,11 +109,7 @@ pcl::registration::DefaultConvergenceCriteria<Scalar>::hasConverged ()
   if (fabs (correspondences_cur_mse_ - correspondences_prev_mse_) / correspondences_prev_mse_ < mse_threshold_relative_)
   {
     if (iterations_similar_transforms_ < max_iterations_similar_transforms_)
```

Comment:
Thanks, I'll deal with it.

Reply:
```suggestion
  if (is_similar)
  {
    // Increment the number of transforms that the thresholds are allowed to be similar
    ++iterations_similar_transforms_;
  }
  else
  {
    // When the transform becomes large, reset.
    iterations_similar_transforms_ = 0;
  }
```
Should I remove brackets in this code too? I wrapped because there are two sentences with comments, but this is logically one sentence.